ext/openssl/ossl_pkey_ec.c

```diff
@@ -668,10 +668,13 @@ static VALUE ossl_ec_group_eql(VALUE a, VALUE b)
     GetECGroup(a, group1);
     GetECGroup(b, group2);
 
-    if (EC_GROUP_cmp(group1, group2, ossl_bn_ctx) == 1)
-       return Qfalse;
+    switch (EC_GROUP_cmp(group1, group2, ossl_bn_ctx)) {
+    case 0: return Qtrue;
+    case 1: return Qfalse;
+    default: ossl_raise(cEC_POINT, "EC_GROUP_cmp");
```

Comment:
This should be `eEC_GROUP` instead of `cEC_POINT`.

Reply:
It looks like there are several other calls with this same error, where ossl_raise() is called with `cEC_POINT`. Should I update those as well?